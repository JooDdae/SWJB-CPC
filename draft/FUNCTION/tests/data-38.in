32
19 3 24 26 6 32 22 28 12 7 9 21 30 4 18 8 10 25 15 2 5 13 17 23 11 14 -1
19 32 5 26 31 10 16 25 30 22 6 18 9 21 13 20 27 3 28 12 15 17 4 2 7 -1
2 17 30 23 24 28 14 11 6 22 10 16 5 21 25 12 32 31 18 27 3 20 13 7 29 -1
22 14 3 25 8 2 15 13 17 9 -1
5 -1
11 10 2 3 26 25 20 27 31 9 -1
24 26 23 9 7 8 32 18 17 25 21 -1
28 11 27 7 8 9 26 25 5 15 4 18 24 12 2 3 17 16 19 31 21 10 20 14 23 22 6 13 32 29 30 -1
27 13 6 12 5 18 11 15 7 25 9 8 20 17 19 30 23 28 21 10 14 4 26 16 -1
17 3 19 13 2 5 22 15 4 7 29 23 20 26 16 30 11 -1
23 2 16 25 27 26 11 -1
10 27 15 23 5 22 29 -1
19 6 8 11 13 12 26 29 7 -1
2 22 4 28 30 15 5 27 20 -1
-1
6 32 31 7 5 16 4 25 22 29 28 14 19 3 27 13 17 26 12 30 8 20 21 11 18 9 -1
17 31 11 7 15 18 22 14 12 23 28 16 9 2 6 -1
9 8 20 26 4 12 16 21 32 29 11 28 27 30 19 2 24 3 -1
28 15 4 -1
32 4 8 5 28 27 19 23 29 15 7 16 14 31 21 26 17 30 18 20 11 25 -1
9 11 28 12 27 15 3 32 6 13 29 16 31 17 25 18 26 10 7 5 19 21 20 22 14 4 23 -1
8 32 13 21 2 -1
32 8 18 10 6 7 4 21 26 22 17 30 16 5 25 27 14 2 19 23 9 28 13 20 3 29 15 24 11 -1
26 29 14 20 30 31 21 23 8 25 10 11 28 6 9 15 19 17 2 3 -1
32 28 21 22 20 29 4 12 14 25 10 5 13 2 6 19 27 9 30 24 26 16 8 7 -1
20 23 2 12 31 17 28 26 18 15 -1
22 11 -1
23 14 10 9 30 29 24 32 31 19 16 20 15 21 11 7 5 17 6 22 8 4 2 25 13 28 26 18 27 3 -1
29 21 6 28 3 17 5 23 10 7 31 20 9 32 13 8 22 -1
16 9 13 5 24 31 19 32 -1
5 20 24 11 8 9 25 10 21 19 17 -1
-1
