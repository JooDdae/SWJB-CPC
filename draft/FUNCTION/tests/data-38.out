YES
8 23 28 